torch/__init__.py

```diff
@@ -1675,7 +1675,7 @@ def compile(model: Optional[Callable] = None, *,
 
     Args:
        model (Callable): Module/function to optimize
-       fullgraph (bool): Whether it is ok to break model into several subgraphs
+       fullgraph (bool): Whether to raise an error on graph breaks. Default is False, i.e. graph breaks are allowed.
```

Comment:
I like the previous description better. The new description adds a notion of graph breaks to the user but we don't quite define it. How about something like the following?
- fullgraph (bool): If False (default), torch.compile attempts to discover compileable regions in the function that it will optimize. If True, then we require that the entire function be capturable into a single graph. If this is not possible (that is, if there are graph breaks), then this will raise an error.